### ISSUE TITLE
Remember user's config on _zen_ mode

### ISSUE THREAD

**Issue by cztuc** [zsviczian/obsidian-excalidraw-plugin#479]
**Is your feature request related to a problem? Please describe.**
After switching to a different file, obsidian-excalidraw resets to default mode, i.e. not honoring _zen_ mode any longer.

**Describe the solution you'd like**
I expect obsidian-excalidraw to remember the user's config: once the user enters _zen_ mode, it stays in _zen_ mode until the user explicitly exists it, no matter if the user changes to a different file.

**Additional context**
I think this will be very useful when the user uses obsidian-excalidraw as a presentation tool.

**Comment by cztuc** [comment]
I just realized that obsidian-excalidraw already allows the user to choose a default mode in the config panel. I'm sorry for not noticing it earlier!

Meanwhile, it still feels buggy when switching between files: the interface goes to normal mode first and then to zen/preview mode (we can see all the components in normal mode for a short while before they completely disappear)

**Comment by zsviczian** [comment]
So there does seem to be a conflict between zen mode, tray mode, and view mode.  Tray mode is my hack on top of Excalidraw, I simply force the Mobile view in desktop mode.  If the user selects zen-mode or view-mode as the default view, I will disable tray-mode.

Note that you can control how files are displayed at a file level. Add `excalidraw-default-mode: view` (or zen, or normal) to the front matter of a drawing. This is useful when you want to use a document as a Map of Content, i.e. normally you do not edit but simply use the page for navigation.

**Comment by cztuc** [comment]
Thanks. `excalidraw-default-mode: view` this is very useful.

**Comment by cztuc** [comment]
> Meanwhile, it still feels buggy when switching between files: the interface goes to normal mode first and then to zen/preview mode (we can see all the components in normal mode for a short while before they completely disappear)

I just want to add a screencast to show what looks `buggy`: the excalidraw toolbar flashes when switching between files. Is it possible to avoid that?

https://user-images.githubusercontent.com/69794131/155387023-dade1f5d-9bd2-4526-983c-bfe4cb4ed040.mp4

In the above screencast, I set `excalidraw-default-mode: view` to both files.

**Comment by zsviczian** [comment]
Yes, I understand it would look nicer if it didn't initialize in normal mode and then switch to view mode. It is not buggy, just there's a sequence of initialization. 

Yes, it could be modified (everything can be modified, just takes time),... BTW, I think this would need to be addressed in the core Excalidraw product and not in the plugin... however in the core package this issue never comes up, since when you close and reopen Excalidraw.com it will always opens in Normal mode.

So in short, I recognize it could be nicer, but frankly, it is not a priority that I plan to work on.

**Comment by zsviczian** [comment]
I fixed this in 1.7.10. Will release it in a couple of days.
